app/polls/onboarding/components/InsightDisplay.js

```diff
@@ -0,0 +1,60 @@
+'use client'
```

Comment:
Please divide these into three individual components.
Some of these components will not need the `use client` and will be server components, which is better for performance.

Reply:
Looking at the code again, none of these should need to be a client component